### PR TITLE
issue #32: added support of the 7.1 php version (changed config files)

### LINKED ISSUE #32: Support of the 7.1 php version?

### PR DESCRIPTION
**Pull request by o1eksiy**
added support PHP 7.1 to config files:
travis.yml
composer.json

This will fix #32

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
         }
     ],
     "require": {
-        "php": "^5.5 || ^7.0",
+        "php": "^5.5 || ^7.0 || ^7.1",
```
Comment (xabbuh) [inline composer.json:18]: this change is not needed, `^7.0` means `>=7.0,<8.0`

Comment (o1eksiy) [reply composer.json:18]: sorry, I didn't know it

Comment (Nyholm) [reply composer.json:18]: Please remove the change on this line and I will be happy to merge your PR.

**Comment by dbu** [comment]
fixed in #35

**Pull request closed without merging**